test/common/pvc_validate.go

```diff
@@ -0,0 +1,83 @@
+package common
+
+import (
+	goctx "context"
+	"fmt"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+//Define list of namespaces and their claims
+var (
+	pvcNamespaces = []PersistentVolumeClaim{
+		{
+
+			Namespace: "redhat-rhmi-fuse",
+			PersistentVolumeClaimNames: []string{
+				"syndesis-meta",
+				"syndesis-prometheus",
+			},
+		},
+
+		{
+
+			Namespace: "redhat-rhmi-middleware-monitoring-operator",
+			PersistentVolumeClaimNames: []string{
+				"prometheus-application-monitoring-db-prometheus-application-monitoring-0",
+			},
+		},
+
+		{
+
+			Namespace: "redhat-rhmi-solution-explorer",
+			PersistentVolumeClaimNames: []string{
+				"user-walkthroughs",
+			},
+		},
+		{
+
+			Namespace: "redhat-rhmi-operator",
+			PersistentVolumeClaimNames: []string{
+				"standard-authservice-postgresql",
+			},
+		},
+	}
+)
+
+func TestPVClaims(t *testing.T, ctx *TestingContext) {
+
+	//get full list of volume claim items
+	pvcs := &corev1.PersistentVolumeClaimList{}
+
+	for _, pvcNamespace := range pvcNamespaces {
+		err := ctx.Client.List(goctx.TODO(), pvcs, &k8sclient.ListOptions{Namespace: pvcNamespace.Namespace})
+		if err != nil {
+			t.Errorf("Error getting PVCs for namespace: %v %w", pvcNamespace.Namespace, err)
+		}
+		for _, claim := range pvcNamespace.PersistentVolumeClaimNames {
+			//check if claim exists
+			_, err := checkForClaim(claim, pvcs)
+			if err != nil {
+				t.Errorf("Persistant Volume Claim: %v %v", claim, err)
+			}
+
+		}
+	}
+}
+
+func checkForClaim(claim string, pvcs *corev1.PersistentVolumeClaimList) (bool, error) {
```

Comment:
```suggestion
func checkForClaim(claim string, pvcs *corev1.PersistentVolumeClaimList) error {
```

As the `bool` result is not used when the function is called, I'd just remove it

Reply:
good call. Updated.